Formula/kops.rb

```diff
@@ -0,0 +1,28 @@
+class Kops < Formula
+  desc "Production Grade K8s Installation, Upgrades, and Management"
+  homepage "https://github.com/kubernetes/kops"
+  url "https://github.com/kubernetes/kops/archive/v1.4.1.tar.gz"
+  sha256 "69b3c9d7e214109cfd197031091ed23963383c894e92804306629f6a32ab324b"
+  head "https://github.com/kubernetes/kops.git"
+
+  depends_on "go" => :build
+  depends_on "kubectl"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    kopspath = buildpath/"src/k8s.io/kops"
+    kopspath.install Dir["*"]
+
+    cd kopspath do
+      system "make"
+    end
+
+    cd buildpath do
```

Comment:
This is unnecessary; you'll be `cd`d back there after you leave the `cd kopspath do` block.